### PR TITLE
Enki bot js

### PR DESCRIPTION
**Pull request by elliehoward**

### CODE REVIEW

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - javascript/core/syntax/javascript-basics-part-2.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - javascript/core/syntax/javascript-basics-part-2.md

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/intro-js/js-practice-console-in.md?ref=de5e901e68f00a877e227cddd82cace33e8a4533
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/intro-js/js-practice-using-javascript-on-a-webpage-inline.md?ref=de5e901e68f00a877e227cddd82cace33e8a4533
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/intro-js/js-practice-using-javascript-on-a-webpage.md?ref=de5e901e68f00a877e227cddd82cace33e8a4533
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/syntax/double-and-single-quotes.md?ref=de5e901e68f00a877e227cddd82cace33e8a4533
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/syntax/javascript-basics-part-2.md?ref=de5e901e68f00a877e227cddd82cace33e8a4533
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/be-careful-when-using-typeof.md?ref=de5e901e68f00a877e227cddd82cace33e8a4533
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/js-practice-booleans.md?ref=de5e901e68f00a877e227cddd82cace33e8a4533
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/js-practice-hoisting.md?ref=de5e901e68f00a877e227cddd82cace33e8a4533
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/js-practice-taking-precedence.md?ref=de5e901e68f00a877e227cddd82cace33e8a4533
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/js-practice-undefined-and-null.md?ref=de5e901e68f00a877e227cddd82cace33e8a4533
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/taking-precedence.md?ref=de5e901e68f00a877e227cddd82cace33e8a4533
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/types-hoisting.md?ref=de5e901e68f00a877e227cddd82cace33e8a4533
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/types-introduction.md?ref=de5e901e68f00a877e227cddd82cace33e8a4533
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/types-string.md?ref=de5e901e68f00a877e227cddd82cace33e8a4533

File: javascript/core/types/js-practice-hoisting.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  - basic
+
+tags:
+
+  - introduction
+
+type: exercise
+
+link: https://jsfiddle.net/elliehoward/0jrhdL7e/
```
Comment (lizTheDeveloper) [inline javascript/core/types/js-practice-hoisting.md]: let's find or create a codewars thing for this

File: javascript/core/types/js-practice-taking-precedence.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  - basic
+
+tags:
+
+  - introduction
+
+type: exercise
+
+link: https://jsfiddle.net/elliehoward/wf31p4hq/
```
Comment (lizTheDeveloper) [inline javascript/core/types/js-practice-taking-precedence.md]: also this

File: javascript/core/types/js-practice-undefined-and-null.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+  - basic
+
+tags:
+
+  - introduction
+
+type: exercise
+
+link: https://jsfiddle.net/elliehoward/em59h4rg/
```
Comment (lizTheDeveloper) [inline javascript/core/types/js-practice-undefined-and-null.md]: also this

**Pull request closed without merging**